packages/api/src/external/commonwell/command/cw-patient-data/update-cw-data.ts

```diff
@@ -58,7 +58,7 @@ export async function updateCwPatientDataWithinDBTx(
   const { data: newData } = update;
   const updatedLinks = [...(newData.links ?? []), ...existing.data.links];
   const uniqueUpdatedLinks = uniqBy(updatedLinks, function (nl) {
-    return nl._links?.self?.href;
+    return nl.patient?.provider?.reference ?? "";
```

Comment:
My concern here is how often does the whole`.patient.provider.reference` is present.

Like, if `.patient` is present a lot but the whole thing not so much, maybe we could have an intermediate fall back before using empty string?

Trying to minimize losing links.

By the way, the reason for this is because we get the same link more than once?

Also, you mentioned something about comparing the hash of the links. Doesn't it work well?

Reply:
So far it's present on 100% of our first 2281 CW links, so I think it works.

The reason for this is that each PD, we get the same links, and so need to deduplicate. Within a single PD I don't think we get the same links more than once.

Reply:
Not related to the hashing of links?

Lastly, why empty string as a fallback? I think we should prefer `undefined` instead of empty string as its a better representation of a missing/empty value.